sdk/appconfiguration/app-configuration/package.json

```diff
@@ -24,6 +24,9 @@
   "browser": {
     "./dist-esm/src/internal/cryptoHelpers.js": "./dist-esm/src/internal/cryptoHelpers.browser.js"
   },
+  "react-native": {
```

Comment:
Are we going to do this for every package?

Reply:
I asked Chris before, the answer was it should be done case by case and we want to remove blockers first. For app-conf there are some issue related to xml API in browsers,  but can be worked around